From: Andreas Herrmann <andreas.herrmann3@amd.com>
Subject: x86, amd-nb: Complete the rename of AMD NB and
References: fate#311968
Patch-mainline: v2.6.38-rc1

Signed-off-by: Thomas Renninger <trenn@suse.de>

Not only the naming of the files was confusing, it was even more so for
the function and variable names.

Renamed the K8 NB and NUMA stuff that is also used on other AMD
platforms. This also renames the CONFIG_K8_NUMA option to
CONFIG_AMD_NUMA and the related file k8topology_64.c to
amdtopology_64.c. No functional changes intended.

Signed-off-by: Hans Rosenfeld <hans.rosenfeld@amd.com>
Signed-off-by: Borislav Petkov <borislav.petkov@amd.com>
Automatically created from "patches.arch/0027-x86-amd-nb-Complete-the-rename-of-AMD-NB-and.patch" by xen-port-patches.py

--- sle11sp2-2011-06-28.orig/arch/x86/kernel/setup-xen.c	2011-06-28 14:01:39.000000000 +0200
+++ sle11sp2-2011-06-28/arch/x86/kernel/setup-xen.c	2011-06-28 14:02:42.000000000 +0200
@@ -816,7 +816,7 @@ static struct dmi_system_id __initdata b
 void __init setup_arch(char **cmdline_p)
 {
 	int acpi = 0;
-	int k8 = 0;
+	int amd = 0;
 
 #ifdef CONFIG_XEN
 	unsigned int i;
@@ -1132,11 +1132,11 @@ void __init setup_arch(char **cmdline_p)
 	acpi_numa_init();
 #endif
 
-#ifdef CONFIG_K8_NUMA
-	k8 = !k8_numa_init(0, max_pfn);
+#ifdef CONFIG_AMD_NUMA
+	amd = !amd_numa_init(0, max_pfn);
 #endif
 
-	initmem_init(0, max_pfn, acpi, k8);
+	initmem_init(0, max_pfn, acpi, amd);
 
 #ifdef CONFIG_ACPI_SLEEP
 	/*
